circe/src/main/scala/org/http4s/circe/CirceInstances.scala

```diff
@@ -25,11 +25,15 @@ trait CirceInstances {
     jsonEncoderWithPrinter(defaultPrinter)
 
   def jsonEncoderWithPrinter(printer: Printer): EntityEncoder[Json] =
-    EntityEncoder[String].contramap[Json] { json =>
-      // Comment from ArgonautInstances (which this code is based on):
-      // TODO naive implementation materializes to a String.
-      // See https://github.com/non/jawn/issues/6#issuecomment-65018736
-      printer.pretty(json)
+    EntityEncoder[Array[Byte]].contramap[Json] { json =>
+      val bytes = printer.prettyByteBuffer(json)
+      if (bytes.remaining < 1) {
```

Comment:
There is no `EntityEncoder[ByteVector]`. I guess it was replaced with `fs2.Chunk`